core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/hashJoin/HashJoinOpDesc.scala

```diff
@@ -0,0 +1,67 @@
+package edu.uci.ics.texera.workflow.operators.hashJoin
+
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonProperty, JsonPropertyDescription}
+import com.google.common.base.Preconditions
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.amber.engine.operators.OpExecConfig
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.{OneToOneOpExecConfig, OperatorDescriptor}
+import edu.uci.ics.texera.workflow.common.operators.filter.FilterOpDesc
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema
+
+class HashJoinOpDesc[K] extends OperatorDescriptor {
+
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Larger table attribute")
```

Comment:
This is confusing.. The user has to decide which of the tables is bigger. If he randomly thinks that one of the tables is bigger, when it's not, then there will be an error